controllers/tweet-controller.js

```diff
@@ -13,6 +14,27 @@ const tweetController = {
     } catch (err) {
       next(err)
     }
+  },
+  addTweet: async (req, res, next) => {
+    const userId = Number(helpers.getUser(req).id)
+    const { description } = req.body
+    if (!userId || !description) {
+      return res.status(400).json({
+        status: 'error',
+        message: 'userId and description required'
+      })
+    }
+
+    if (description.length > 140) {
+      return res.status(400).json({
+        status: 'error',
+        message: 'tweet should be within 140 characters'
+      })
+    }
+
+    const tweet = await Tweet.create({ userId, description })
+    return res.status(200).json(tweet.toJSON())
```

Comment:
The "res.json()" method sends a JSON response. I think we don't need to use .toJSON() to convert the object before sending it to .json() method as its parameter.

Reply:
Oh~ I forgot that one. Used to do this the whole semester 3 😂